codalab/worker/local_run/docker_image_manager.py

```diff
@@ -102,13 +104,10 @@ def _cleanup(self):
                     entry_to_remove.digest,
                 )
                 try:
-                    image_to_delete = self._docker.images.get(entry_to_remove.id)
-                    tags_to_delete = image_to_delete.tags
-                    for tag in tags_to_delete:
-                        self._docker.images.remove(tag)
-                    # if we successfully removed the image also remove its cache entry
+                    # We try to delete images from image cache in LRU order
+                    self._docker.images.remove(entry_to_remove.id)
```

Comment:
Can you explain in this PR what this change does?

Reply:
The original code tries to delete the image by looping through and deleting all the tags associated with the image one by one. I found a simpler API when I read through rest api's documentation: given that we already have `image.id` in the variable `entry_to_remove`, we can directly remove the image using `image.id`.